pkgs/tools/admin/okta-aws-cli/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "okta-aws-cli";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner  = "okta";
+    repo   = "okta-aws-cli";
+    rev    = "v${version}";
+    sha256 = "1qlc04fydcqnifbhrzdw63wq3ndzsyzp4s1bdvfhvizbhdi03mpp";
+  };
+
+  vendorSha256 = "1gzaadpf9zrci7yv88f434mvc7gralb7dj6wl6r4vsv3qk291680";
+
+  doCheck = false;
+
+  ldflags = [ "-s" "-w" "-X=main.Version=${version}" ];
```

I don't think this is working properly

```
 ./result/bin/okta-aws-cli --version
okta-aws-cli version 1.0.0
```

It's supposed to be 1.0.1.

It looks like upstream they have this configured in an [internal package as a const](https://github.com/okta/okta-aws-cli/blob/08b67cda2363c824c6a10ab9665f6844714cddda/internal/config/config.go#L35) which means we can't change it from the build process. I recommend you file a PR upstream to change this to a variable and update the package in nixpkgs at a later time. I can guide you privately on how to do that if you're unfamiliar with Go.

```suggestion
  ldflags = [ "-s" "-w" ];
```